ui_framework/components/expression/_expression.scss

```diff
@@ -1,107 +1,107 @@
-$expressionColorHighlight: #00A69B;
+.kuiExpressionItem {
+  display: inline-block;
+  position: relative;
 
-.kuiExpression {
-
-  .kuiExpression__item {
-    display: inline-block;
-    margin-right: 10px;
-    position: relative;
+  & + & {
+    margin-left: 10px;
   }
+}
 
-  .kuiExpression__button {
-    background-color: transparent;
-    padding: 5px 0px;
-    border: none;
-    border-bottom: dotted 2px $globalBorderColor;
-    font-size: $globalFontSize;
-    cursor: pointer;
-
-    .kuiExpression__buttonDescription {
-      color: $expressionColorHighlight;
-      text-transform: uppercase;
-    }
+.kuiExpressionItem__button {
```

Comment:
Can we indent this class and all of the rest of the classes once? Reasoning is here: https://github.com/elastic/kibana/blob/master/style_guides/css_style_guide.md#create-descendant-classes-to-represent-child-components-which-cant-stand-on-their-own